subworkflows/local/utils_nfcore_panoramaseq_pipeline/main.nf

```diff
@@ -74,21 +74,17 @@ workflow PIPELINE_INITIALISATION {
 
     Channel
         .fromList(samplesheetToList(params.input, "${projectDir}/assets/schema_input.json"))
-        .map {
-            meta, fastq_1, fastq_2 ->
-                if (!fastq_2) {
-                    return [ meta.id, meta + [ single_end:true ], [ fastq_1 ] ]
-                } else {
-                    return [ meta.id, meta + [ single_end:false ], [ fastq_1, fastq_2 ] ]
-                }
-        }
-        .groupTuple()
-        .map { samplesheet ->
-            validateInputSamplesheet(samplesheet)
-        }
-        .map {
-            meta, fastqs ->
-                return [ meta, fastqs.flatten() ]
+        .map { row ->
+            def meta = row[0]
+            def R1 = row[1]
+            def R2 = row[2]
```

Comment:
```suggestion
        .map { meta, R1, R2 ->
```

You can directly assign variable names when a list has been given as input to the `.map` operators